autokeras/hypermodels/wrapper.py

```diff
@@ -200,3 +217,497 @@ class GeneralBlock(block_module.Block):
 
     def build(self, hp, inputs=None):
         raise NotImplementedError
+
+
+class SegmentationBlock(block_module.Block):
+    """Block for image segmentation.
+
+    The image blocks is a block choosing from ResNetBlock, XceptionBlock, ConvBlock,
```

Comment:
Please add a link to the paper we are using like what we did in the docstrings of XceptionBlock.